### PR TITLE
Description of this test assertion implies that we should use not equals

### PR DESCRIPTION
**Pull request by calvinchengx**
Resolves https://github.com/openwebf/webf/issues/362 if approved.

### CODE REVIEW

File: bridge/test/webf_test_context.cc [hunk 1]
```diff
@@ -367,7 +367,7 @@ void WebFTestContext::registerTestEnvDartMethods(uint64_t* methodBytes, int32_t
   dartMethodPtr->simulatePointer = reinterpret_cast<SimulatePointer>(methodBytes[i++]);
   dartMethodPtr->simulateInputText = reinterpret_cast<SimulateInputText>(methodBytes[i++]);
 
-  assert_m(i == length, "Dart native methods count is not equal with C++ side method registrations.");
```
Comment (andycall) [inline bridge/test/webf_test_context.cc:370]: This assertion is correct; it checks that the count of registered Dart methods and the initialized member of DartPointer structs are explicitly equal.

If it fails, it indicates that the functions registered on the Dart side do not match with those in C++, and this mismatch could lead to unexpected behavior.

In this case, the mocked Dart functions do not match with those in C++ in the unit test environment. We need to add additional mock functions to fix it.

Comment (calvinchengx) [reply bridge/test/webf_test_context.cc:370]: Thanks. Yep. It was either changing the assertion, or adding this patch that you have added in https://github.com/openwebf/webf/pull/366/commits/51d35e37d55ca37e4a93d8718664e7a3d755ce47

**Review by answer-huang — APPROVED** [review]

File: bridge/test/webf_test_context.cc (hunk 1 above)
Comment (andycall) [reply bridge/test/webf_test_context.cc:370]: @calvinchengx Please revert the changes in e136cd0cc65c1b1e86d83ca0ceb45f9d2988e755

**Comment by calvinchengx** [comment]
Got it. Updated.

www.calvinx.com ( http://www.calvinx.com )
Stalk me! @ www.twitter.com/calvinchengx ( http://www.twitter.com/calvinchengx )

**Review by XGHeaven — APPROVED** [review]

**Pull request merged**